common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -122,5 +122,17 @@
   "feature.wynntils.tooltipScale.universalScale.name": "Universal Scale",
   "feature.wynntils.userFeature.userEnabled.description": "Should this feature be enabled?",
   "feature.wynntils.userFeature.userEnabled.name": "Enabled",
-  "feature.wynntils.wynncraftButton.name": "Wynncraft Button Feature"
+  "feature.wynntils.wynncraftButton.name": "Wynncraft Button Feature",
+  "overlay.wynntils.overlay.horizontalAlignmentOverride.description": "Set this if you want to override the position horizontal alignment",
```

Comment:
Right, now I understand (I think). For translation keys, we want the key to be unique per "kind" of configuration for that overlay. But when storing config for these values, we need a config json key that is unique per instance of overlay.

I got confused since for features we have had a 1-to-1 relationship between "kinds" of features and instances of them.

Reply:
Yeah for config values in abstract classes, we only need to define per superclass, not per every implementation.